docs/deploy/FhirConverter-SingleAzureDeploy.bicep

```diff
@@ -93,12 +93,27 @@ param keyVaultUserAssignedIdentityName string = '${serviceName}-kv-identity'
 @description('If set to true, security requirements will be enabled on the API endpoint. This is strongly recommended.')
 param securityEnabled bool = false
 
-@description('Audiences for the api authentication.')
+@description('Audiences for the api authentication. Only applicable when securityEnabled is set to true.')
 param securityAuthenticationAudiences array = []
 
-@description('Authority for the api authentication.')
+@description('Authority for the api authentication. Only applicable when securityEnabled is set to true.')
 param securityAuthenticationAuthority string = ''
 
+@description('When set to true, the template Storage Account will only accept network traffic from within the specified Virtual Network, which the Container Apps environment will be onboarded to.')
+param storageAccountNetworkIsolationEnabled bool = false

```

Comment:
nit: should this be a generic param like the one you have defined in other templates 'configureNetworkIsolation' ? Since the VNet if created is tied to multiple resources ACA env, storage account, etc.

Reply:
What that said, this has me thinking about the option of isolating everything in the infrastructure (i.e. key vault, etc.) on the storage account. It's not part of our security requirement, so maybe not high priority, but do you think that would be worthwhile?

Reply:
No, I don't think we should address that atm if it is not being driven by security wave requirements currently.